components/binding-usage-controller/docs/usage-kind.md

```diff
@@ -0,0 +1,33 @@
+# UsageKind
+## Overview
+
+UsageKind allows dynamic, in the runtime, Binding Usage Controller configuration. It allows you to define what resources can be bound with ServiceBinding and how to bind them.
+
+## UsageKind structure
+
+From the technical point of view the binding process modifies target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod and the resource type (group, kind and version). 
+
+## Binding Usage Controller
+
+Binding Usage Controller takes the value of the spec.usedBy.kind field of the binding usage resource, looks for the corresponding usage kind which contains information about the resource to be bound. The controller handles resources even if the specified labelsPath field does not exist.
```

Comment:
``The Binding Usage Controller takes the value of the **spec.usedBy.kind** field of the ServiceBindingUsage resource and looks for the corresponding UsageKind which contains information about the resource to be bound. The controller handles resources even if the specified **labelsPath** field does not exist.``